docs/spelling_wordlist.txt

```diff
@@ -718,6 +720,7 @@ faas
 facebook
 failover
 faq
+favour
```

Comment:
Actually -- we should be consistent in our spellings, and not mix en_GB and en_US